From 7d9e69664351fc312d1d4ea99ec787888f8caee3 Mon Sep 17 00:00:00 2001
From: Everett Hildenbrandt <hildenb2@illinois.edu>
Date: Sat, 23 Sep 2017 10:32:01 -0500
Subject: [PATCH] src/.../Makefile.am: add local include directory

-   CVC4 is now built and its includes are placed at `.build/local/include`, so
    that's added to the CPPFLAGS automake variables where needed.
---
 src/Meta/Makefile.am   | 3 ++-
 src/Mixfix/Makefile.am | 3 ++-
 src/SMT/Makefile.am    | 3 ++-
 3 files changed, 6 insertions(+), 3 deletions(-)

diff --git a/src/Meta/Makefile.am b/src/Meta/Makefile.am
index b4049051..32ce1f52 100755
--- a/src/Meta/Makefile.am
+++ b/src/Meta/Makefile.am
@@ -18,7 +18,8 @@ libmeta_a_CPPFLAGS = \
 	-I$(top_srcdir)/src/IO_Stuff \
 	-I$(top_srcdir)/src/StrategyLanguage \
 	-I$(top_srcdir)/src/3rdParty \
-	-I$(top_srcdir)/src/SMT
+	-I$(top_srcdir)/src/SMT \
+	-I$(top_srcdir)/.build/local/include
 
 libmeta_a_SOURCES = \
 	metaModuleCache.cc \
diff --git a/src/Mixfix/Makefile.am b/src/Mixfix/Makefile.am
index acf25c49..39d64035 100755
--- a/src/Mixfix/Makefile.am
+++ b/src/Mixfix/Makefile.am
@@ -29,7 +29,8 @@ libmixfix_a_CPPFLAGS = \
 	-I$(top_srcdir)/src/3rdParty \
 	-I$(top_srcdir)/src/FullCompiler \
 	-I$(top_srcdir)/src/StrategyLanguage \
-	-I$(top_srcdir)/src/SMT
+	-I$(top_srcdir)/src/SMT \
+	-I$(top_srcdir)/.build/local/include
 
 libmixfix_a_SOURCES = \
 	syntaxContainer.cc \
diff --git a/src/SMT/Makefile.am b/src/SMT/Makefile.am
index 4b3de566..17021873 100755
--- a/src/SMT/Makefile.am
+++ b/src/SMT/Makefile.am
@@ -8,7 +8,8 @@ libSMT_a_CPPFLAGS = \
 	-I$(top_srcdir)/src/FullCompiler \
 	-I$(top_srcdir)/src/NA_Theory \
 	-I$(top_srcdir)/src/FreeTheory \
-	-I$(top_srcdir)/src/BuiltIn
+	-I$(top_srcdir)/src/BuiltIn \
+	-I$(top_srcdir)/.build/local/include
 
 libSMT_a_SOURCES = \
 	SMT_Symbol.cc \
-- 
2.13.5

